src/test/java/jenkins/plugins/logstash/persistence/BuildDataTest.java

```diff
@@ -198,37 +183,14 @@ public void constructorSuccessBuiltOnSlave() throws Exception {
     Assert.assertEquals("Incorrect buildHost", "Test Slave 01", buildData.getBuildHost());
     Assert.assertEquals("Incorrect buildLabel", "Test Slave", buildData.getBuildLabel());
 
-    verify(mockBuild).getId();
-    verify(mockBuild, times(2)).getResult();
-    verify(mockBuild, times(2)).getParent();
-    verify(mockBuild, times(2)).getDisplayName();
-    verify(mockBuild).getFullDisplayName();
-    verify(mockBuild).getDescription();
-    verify(mockBuild).getUrl();
-    verify(mockBuild).getAction(AbstractTestResultAction.class);
-    verify(mockBuild).getBuiltOn();
-    verify(mockBuild, times(2)).getNumber();
-    verify(mockBuild).getTimestamp();
-    verify(mockBuild, times(3)).getRootBuild();
-    verify(mockBuild).getBuildVariables();
-    verify(mockBuild).getSensitiveBuildVariables();
-    verify(mockBuild).getEnvironments();
-    verify(mockBuild).getEnvironment(mockListener);
-
-    verify(mockTestResultAction).getTotalCount();
-    verify(mockTestResultAction).getSkipCount();
-    verify(mockTestResultAction).getFailCount();
-    verify(mockTestResultAction, times(1)).getFailedTests();
-
-    verify(mockProject, times(2)).getName();
-
-    verify(mockDate).getTime();
+    verifyMocks();
+    verifyTestResultActions();
   }
 
   @Test
   public void constructorSuccessTestFailures() throws Exception {
     TestResult mockTestResult = Mockito.mock(hudson.tasks.test.TestResult.class);
-    when(mockTestResult.getSafeName()).thenReturn("Mock Test");
+    //when(mockTestResult.getSafeName()).thenReturn("Mock Test");
```

Comment:
Please don't leave commented out code

If this method is no longer called please just remove it

Reply:
removed